lambdas/data-migration2/tests/test-granulesAndFiles.js

```diff
@@ -488,8 +506,40 @@ test.serial('migrateGranulesAndFiles skips already migrated granule record', asy
     },
   });
 
-  const records = await knex(tableNames.granules);
+  const records = await t.context.granulePgModel.search(t.context.knex, {});
+  t.is(records.length, 1);
+
+  t.teardown(async () => {
+    await t.context.granulePgModel.delete(t.context.knex, { cumulus_id: records[0].cumulus_id });
+  });
+});
+
+test.serial('migrateGranulesAndFiles processes granule with no files', async (t) => {
```

Comment:
I noticed that this wasn't previously supported and it seems like a possible scenario since `files` aren't a required property of granules in Dynamo

Reply:
Do you mean "covered" or "supported"?